helm/defectdojo/templates/sa.yaml

```diff
@@ -0,0 +1,16 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: defectdojo
```

Comment:
Can you rename it, that it is consistent with the other templates (i.e. {{ $fullName }}-service-account). Otherwise we may get problem in environments with multiple deployments.

Reply:
fixed